example/demo/src/index.ts

```diff
@@ -11,3 +9,10 @@ const models = { ...inventory, ...controls, ...delivery }
 const port = Number(process.env.PORT || '3005')
 
 createWSServer(models, port)
+
+import '@rise-tools/kit-react-navigation/server'
+declare module '@rise-tools/kit-react-navigation/server' {
+  interface Navigate {
+    screens: Record<keyof typeof models, void>
```

Comment:
can add this in `template` too.

Reply:
Template currently doesn't have any screens, so there's no point. But if we had two screens and navigate action, it would make sense. We may need to create additional template soon!